docs/program/extend/SDK-as-server.md

```diff
@@ -1,31 +1,21 @@
 ---
 
-Viam's Robot Development Kit ([RDK](/internals/rdk/)) provides a variety of components and services out of the box.
-If the components you are working with are not supported, you can use Viam [modular resources](/program/extend/) to add support for custom resources which form a seamless extension of the Viam platform.
+If a type or model of [component](/components) you are working with is not built into the [Viam RDK](/internals/rdk), you can use a [Viam SDK](/program/sdk-as-client) to code a custom resource implementation, host it on a server, and add it as a [remote](/manage/parts-and-remotes) of your robot.
+Then, you can configure a command to launch this remote server as a [process](/appendix/glossary/#term-process) of your robot to make sure the remote server is always running alongside the rest of your robot.
 
-Currently, [modular resource](/program/extend/) development is supported only with the [RDK (Go SDK)](https://pkg.go.dev/go.viam.com/rdk).
-However, you can leverage any Viam [SDK](/program/extend/sdk-as-server) to create a custom component implementation by using an SDK as a server.
+After configuring the remote server, you can control and monitor your component programmatically with the SDKs and from the [Viam app](https://app.viam.com/).
 
-## What is a custom component implementation?
-
-Implementing a custom component allows you to create a new component type (or a new model of an existing type, such as a new `arm` model) that interfaces with `viam-server` using Viam's SDKs.
-A custom component is a subclass of a [component](https://python.viam.dev/autoapi/viam/components/component_base/index.html#module-viam.components.component_base) that allows you to register the new component with the `viam-server` as a {{< glossary_tooltip term_id="remote" text="remote" >}} after you spin up an SDK server (which can have one or many custom components).
-
-In other words, a custom component implementation is a way for you to integrate unsupported components into a robot running on the Viam platform.
-
-## Why use the SDK to create a custom component implementation?
-
-While the main Viam RDK is written in Go, you can create custom components using the Viam SDKs (like Python) and connect them to a robot as a remote component.
-This allows you to use hardware that is not natively supported by Viam.
-Once you create and register your custom component with `viam-server`, you will be able to control and monitor your component from the Viam SDK and from the [Viam app](https://app.viam.com/).
-
-## What is required to create a custom component?
+This option makes reconfiguration more difficult than programming custom [modular resources](/program/extend/modular-resources), and is a less seamless integration.
```

Comment:
So, why would anyone go this route?  Also, this sentence is confusing - I don't think you mean that if you go this route that on a scale reconfiguration is harder than programming  modular resources.

Reply:
That's a good point, I think I will just delete the sentence, but will make an update in a similar vein when this option is fully deprecated/modular resource registry support is more ready